firebase-crashlytics/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
           package="com.google.firebase.crashlytics" >
 
     <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-sdk android:minSdkVersion="16"/>
```

Comment:
Can we add a comment that this is for non-gradle builds, a la https://github.com/firebase/firebase-android-sdk/blob/master/firebase-database/src/main/AndroidManifest.xml#L19 ?

Reply:
Done! Thanks for catching that! (Also I mistakenly looked at post copybara change :) (copybara uncomments <uses-sdk android:minSdkVersion="16"/>)

Reply:
Thanks for fixing that up! :)